src/outputProcessor/TextOutputProcessor.js

```diff
@@ -11,8 +11,7 @@ module.exports = class TextOutputProcessor extends BaseOutputProcessor {
     this.chunkSeparator = EOL + EOL;
   }
 
-  write(allMessages) {
-
+  write(/** @type {import('../installLogsPrinter').AllMessages} */ allMessages) {
```

Comment:
What is the purpose of this kind of typing effort? If we would be going this path I would rather just start transforming the whole repo into typescript.

Reply:
> What is the purpose of this kind of typing effort?

It's a way to specify a type in javascript for VScode. For example here, you get the types, rather than `messages` being `any`.
![image](https://github.com/archfz/cypress-terminal-report/assets/87083504/befa6819-11ca-48b5-819a-2d8e7c54e6ed)

vs.

![image](https://github.com/archfz/cypress-terminal-report/assets/87083504/d7b043df-d0f3-4750-9de3-38f4b8c6db7b)

I get that these comments are ugly though so if you want me to revert those, I'm totally down with that.

>  If we would be going this path I would rather just start transforming the whole repo into typescript.

That would be optimal compared to maintaining the `.d.ts` files, and could help with future debug. I don't think it would be that challenging of a task either. I'll leave that decision up to you though.

Reply:
Lets leave it like this for now. Better than nothing. However in a future pr instead we should invest in the typescript option.